src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue

```diff
@@ -65,34 +69,40 @@
     { label: 'does not exist', value: 'does_not_exist' }
   ])
 
+  const showLabelApplicationAccelerator = computed(() => {
+    if (props.showRequiresApplicationAcceleratorInDescription) return ''
+
+    return ' - requires Application Accelerator'
```

Comment:
sugestão (não bloqueante): dá pra simplificar sem atrapalhar a legibilidade.

```suggestion
    return props.hideApplicationAcceleratorInDescription ? '' :  ' - Requires Application Accelerator'
```

Reply:
não fica bom, pois ultrapassa o limite de caracteres e vai que quebrar a linha e não fica muito bom, prefiro assim mesmo